### PR TITLE
Feature/applications admin

### PR DESCRIPTION
**Pull request by djrobstep**

### CODE REVIEW

File: src/bundles/ApplicationsAdmin/redux/modules/__tests__/applications.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    type: CONVERTED_SELLER,
+    id: 1
+  };
+  expect(convertedSeller(1)).toEqual(expectedAction);
+});
+
+test('default reducer', () => {
+  expect(reducer()).toEqual({});
+  expect(reducer({})).toEqual({});
+  expect(reducer({ foo: 'bar' })).toEqual({ foo: 'bar' });
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/redux/modules/__tests__/applications.test.js]: Can you add a test for `CONVERTED_SELLER` action?

File: src/bundles/ApplicationsAdmin/components/AppList/AppList.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      dispatch(convertApplicationToSeller(id))
+    }
+  }
+}
+
+// const state = {
+//   applications: {}
+//   meta: {},
+//   options: {}
+// }
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/components/AppList/AppList.js]: Delete this

File: src/bundles/ApplicationsAdmin/components/AppList/AppList.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+// function convertApplicationToSeller(id) {
+//   console.log('convertApplicationToSeller');
+//   return {
+//     type: CONVERT_TO_SELLER,
+//     id
+//   }
+// }
+// const convertApplicationToSeller = (id) => {
+//   console.log(id)
+// }
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/components/AppList/AppList.js]: Delete this

File: src/bundles/ApplicationsAdmin/components/AppList/AppList.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { convertApplicationToSeller } from '../../redux/modules/applications'
+
+const AppList = ({meta = {}, applications, onClick}) => (
+  <div><h2>{meta.heading}</h2>
+  <table style={{margin: '0 auto'}}><tbody>
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/components/AppList/AppList.js]: Would prefer to address style via the grid or something similar over in-line.

File: src/bundles/ApplicationsAdmin/redux/modules/applications.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      headers: {
+        // Flask expects the token as a header.
+        'X-CSRFToken': state.form_options.csrf_token
+      }
+    })
+    .then((response) => response.json())
+    .then((json) => {
+      dispatch(convertedSeller(id))
+    })
+    // .then(() => dispatch(postSubmit()));
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/redux/modules/applications.js]: Delete this

File: src/bundles/ApplicationsAdmin/redux/modules/reducer.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux'
+// import { createForms } from 'react-redux-form';
+
+import applications from './applications';
+// import form_options from '../../../../shared/reduxModules/form_options';
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/redux/modules/reducer.js]: Delete this

File: src/bundles/ApplicationsAdmin/redux/modules/reducer.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux'
+// import { createForms } from 'react-redux-form';
```
Comment (nathf) [inline src/bundles/ApplicationsAdmin/redux/modules/reducer.js]: Delete this

File: src/bundles/ApplicationsAdmin/components/AppList/AppList.js [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  </tr>
+  {applications.map((a, i) =>
+    <tr key={a.id}>
+      <td>{a.name}</td>
+      <td>{a.id}</td>
+      <td>{a.email}</td>
+      <td>{a.status}</td>
+      <td>{ a.status === 'submitted' &&
+        <input onClick={e => {
+                console.log('convertApplicationToSeller:' + a.id);
```
Comment (matt-sm) [inline src/bundles/ApplicationsAdmin/components/AppList/AppList.js]: remove the console.log?

**Pull request merged**